circle.yml

```diff
@@ -39,3 +39,23 @@ deployment:
           -l "${CIRCLE_ARTIFACTS}"/acceptance_deploy 2>&1 >
           "${CIRCLE_ARTIFACTS}/acceptance_deploy.log"
       - build/push-aws.sh
+  osx:
+    branch: master
+    commands:
+      - |
+          if [ -n "${BUILD_OSX-}" ]; then
+            set -eux
+            # There must certainly be a better way to move ~/cockroach
```

Comment:
We move it only so that it's in the correct location in GOPATH. We do reuse the source from tests we've already run - those mount `~/cockroach` where I'm moving it to, and the rest of GOPATH is mounted in separately, so it already gets a bootstrapped setup.

I had briefly looked at running it inside of the builder, but then we need the builder to be docker-in-docker as @tamird said, plus we need to run the builder in privileged mode (which looked like CircleCI potentially allowed it after explanation of the use case). But that seems too complicated for its own sake.

Could we just move `~/cockroach` to that location in the setup phase?